packages/@react-types/contextualhelp/src/index.d.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps} from '@react-types/shared';
+import {OverlayTriggerProps, PositionProps} from '@react-types/overlays';
+import {ReactNode} from 'react';
+
+export interface ContextualHelpProps {
+  /** The title of the Contextual Help. */
+  title?: ReactNode
+}
+
+export interface AriaContextualHelpProps extends ContextualHelpProps, DOMProps, AriaLabelingProps {}
```

Comment:
This type isn't being used and may not be needed since this component is already using `ActionButton`, `Dialog` and `DialogTrigger` internally.